tsl/test/expected/continuous_aggs_ddl-11.out

```diff
@@ -798,7 +800,7 @@ SELECT * FROM drop_chunks_view ORDER BY time_bucket DESC;
           10 |  14
            5 |   9
            0 |   4
-(8 rows)
+(10 rows)
```

Comment:
Also didn't expect any changes in output here or below.

Reply:
that cagg didnt get its parameters changed and was already using the new behaviour